tests/conftest.py

```diff
@@ -555,6 +555,46 @@ def mocked_setup_sub_logger(mocker):
         'bigchaindb.log.setup.setup_sub_logger', autospec=True, spec_set=True)
 
 
+@pytest.fixture
+def abci_http(tendermint_host, tendermint_port):
+    import requests
+    import time
+
+    for i in range(5):
+        try:
+            uri = 'http://{}:{}/abci_info'.format(tendermint_host, tendermint_port)
+            requests.get(uri)
+            return True
+
+        except Exception:
+            pass
+        time.sleep(1)
+
+    return False
+
+
+@pytest.yield_fixture(scope='session')
+def event_loop(request):
```

Comment:
`pytest-asyncio` already includes an [`event_loop` fixture](https://github.com/pytest-dev/pytest-asyncio#fixtures).

Reply:
Yes, but to set the scope of `event_loop` to `'session'` it has to be re-defined.